simulator/__tests__/cross.spec.ts

```diff
@@ -65,6 +65,7 @@ describe("cross contract calls", () => {
 
   test("getting view with arg", () => {
     const res = words.view("reverse", { word: { text: "hello" } });
+    console.log(JSON.stringify(res, null, 2));
```

Comment:
nit: extra `console.log`

Reply:
Thanks! Next time can you use the suggestion feature?

Reply:
```suggestion
```